src/service/resources-impl.js

```diff
@@ -169,6 +173,7 @@ export class Resources {
     });
     this.viewport_.onScroll(() => {
       this.lastScrollTime_ = timer.now();
+      this.framerate_.collect();
```

Comment:
Flush should be called by the Performance itself, right? But seems like certainly not in this class.